src/BrazeKit-dev.js

```diff
@@ -248,29 +314,8 @@ var constructor = function () {
     function processEvent(event) {
         var reportEvent = false;
 
-        if (
-            event.EventDataType == MessageType.Commerce &&
-            event.EventCategory == mParticle.CommerceEventType.ProductPurchase
-        ) {
-            reportEvent = logPurchaseEvent(event);
-        } else if (event.EventDataType == MessageType.Commerce) {
-            var listOfPageEvents =
-                mParticle.eCommerce.expandCommerceEvent(event);
-            if (listOfPageEvents != null) {
-                for (var i = 0; i < listOfPageEvents.length; i++) {
-                    // finalLoopResult keeps track of if any logBrazeEvent in this loop returns true or not
-                    var finalLoopResult = false;
-                    try {
-                        reportEvent = logBrazeEvent(listOfPageEvents[i]);
-                        if (reportEvent === true) {
-                            finalLoopResult = true;
-                        }
-                    } catch (err) {
-                        return 'Error logging page event' + err.message;
-                    }
-                }
-                reportEvent = finalLoopResult === true;
-            }
+        if (event.EventDataType == MessageType.Commerce) {
```

Comment:
Is there a good reason to not use triple equals?

```suggestion
        if (event.EventDataType === MessageType.Commerce) {
```